test/e2e_node/cpumanager_test.go

```diff
@@ -221,6 +230,12 @@ var _ = SIGDescribe("CPU Manager", ginkgo.Ordered, framework.WithSerial(), featu
 			cpuCount := 1
 			reservedCPUs := cpuset.New(0)
 
+			cpuReq := int64(cpuCount + reservedCPUs.Size() + 1)   // note the extra for the non-gu pod
+			cpuDetails = cpuDetailsFromNode(getLocalNode(ctx, f)) // avoid potentially costly updateKubeletConfigIfNeeded
+			if cpuDetails.Allocatable < cpuReq {
+				e2eskipper.Skipf("Skipping CPU Manager tests since the CPU allocatable %d < CPU request %d (online %d)", cpuDetails.Allocatable, cpuReq, onlineCPUs.Size())
+			}
+
```

Comment:
WDYT about adding a Ginkgo By(...) description before each skip so it’s easier to trace the reason in test logs?

Reply:
done!